Gemfile

```diff
@@ -2,3 +2,5 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in govuk_tech_docs.gemspec
 gemspec
+
+gem 'middleman-search', :git => "git://github.com/alphagov/middleman-search.git"
```

Comment:
We could add instructions to add this gem to `docs/configuration.md`. We might keep even want to keep and not require `middleman-search` by default so sites that don't use search don't need to download it.

Reply:
Ok, adding this line to their tech docs gemfile would be another step to activating search? It feels like something to avoid. I don't expect tech writers to understand gemfiles and what they do, it adds another step that could break the build if done incorrectly.  It's your call.

Reply:
🚨Adding this line is now a requirement to activating search.